packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -671,6 +671,15 @@ export class UnfurlService extends BaseService {
                     }
 
                     if (lightdashPage === LightdashPage.DASHBOARD) {
+                        // Wait for markdown tiles specifically
+                        const markdownTiles = await page
+                            .locator('.markdown-tile')
+                            .all();
+                        await Promise.all(
+                            markdownTiles.map((tile) =>
+                                tile.waitFor({ state: 'attached' }),
```

Comment:
"attached" is better than "visible" - tile could be empty, so if there's no content it wouldn't be considered as "visible"